app/scripts/filters/canI.js

```diff
@@ -2,15 +2,22 @@
 
 angular
   .module('openshiftConsole')
-  .filter('canIDoAny', function(canIFilter) {
-    var resourceRulesMap = {
+  .filter('canIDoAny', function(APIService, canIFilter) {
+    // Top level keys are representing pages.
+    // Within each page may one or more resources.
+    // canIDoAny returns truthy if the user passes the canIFilter
+    // for at least one of the resources listed in the page.
+    // TODO: this is confusing, we should come back & fix
```

Comment:
I think having an issue is enough. Would rather not clutter the code with TODO comments for things we're already tracking in issues (#2114).

Reply:
agree, will remove